Project.toml

```diff
@@ -14,14 +14,14 @@ StaticArrays = "90137ffa-7385-5640-81b9-e52037218182"
 UnicodePlots = "b8865327-cd53-5732-bb35-84acbb429228"
 
 [compat]
-julia = "^1.5"
 AbstractPlotting = "^0.18.2"
 DiffResults = "^1.0.3"
 ForwardDiff = "^0.10.18"
 GLMakie = "^0.2.9"
 Plots = "^1.13.2"
 StaticArrays = "^1.1.0"
 UnicodePlots = "^1.3.0"
+julia = "^1.5"
```

Comment:
Does the order matter in the `[compat]` block?

Reply:
To be honest I don't know, but my guess is that the order does not matter